### PR TITLE
Update our HipChat docs

### PR DESCRIPTION
**Pull request by rafaelks**

### CODE REVIEW

File: administrator-guides/import/hipchat/cloud/README.md [hunk 1]
```diff
@@ -2,4 +2,9 @@
 
 To import your current data from HipChat Cloud into Rocket.Chat, please do the following.
 
-TODO!
\ No newline at end of file
+1. Export date from HipChat
+2. Wait until your download is ready and download it (it will be a .zip file)
+3. Go to `https://[your_rocket_chat].rocket.chat/admin/import`
```
Comment (geekgonecrazy) [inline administrator-guides/import/hipchat/cloud/README.md]: should include something like and click the hipchat importer

**Comment by rafaelks** [comment]
@geekgonecrazy done

File: administrator-guides/import/hipchat/cloud/README.md [hunk 2]
```diff
@@ -2,4 +2,9 @@
 
 To import your current data from HipChat Cloud into Rocket.Chat, please do the following.
 
-TODO!
\ No newline at end of file
+1. Export date from HipChat
```
Comment (cardoso) [inline administrator-guides/import/hipchat/cloud/README.md]: Export data?

**Comment by rafaelks** [comment]
@cardoso good finding, done

**Review by MartinSchoeler — APPROVED** [review]
Looks good to me

**Pull request merged**